### DISCUSSION TITLE
Smart Campus Navigating System

### DISCUSSION THREAD

**Discussion by akweijudesackey003** [akweijudesackey003/Smart-Campus-Navigating-System.#1] [Ideas]
Guys how we go by it, please let start something

**Comment by Queneee** [comment]
Yes please

**Comment by Queneee** [comment]
Everyone was given what to do right

**Comment by Queneee** [comment]
So with what you've done so far let each and everyone post it here and let's all review it.

**Comment by Queneee** [comment]
Please I've shared with you the purpose of the project we all chose

**Comment by akweijudesackey003** [comment]
Okay, i have talked to Edwin he will aid me to be able to get better understanding and continue the work

**Comment by Derrickheistman** [comment]
okay
